scripts/build_avado.sh

```diff
@@ -58,16 +58,23 @@ if [[ -z $(grep -E "${default_development_environment}" "./build/Dockerfile") ]]
   exit 1
 fi
 
+# Create backups
+cp ./docker-compose.yml ./docker-compose.bak
+cp ./dappnode_package.json ./dappnode_package.bak
+cp ./build/Dockerfile ./build/Dockerfile.bak
+
 # Write AVADO docker build version
 sed -e "s/image:[ ]'hopr\.avado\.dnp\.dappnode\.eth:[0-9]\{1,\}\.[0-9]\{1,\}\.[0-9]\{1,\}/image: 'hopr.avado.dnp.dappnode.eth:${AVADO_VERSION}/" ./docker-compose.yml \
   > ./docker-compose.yml.tmp && mv ./docker-compose.yml.tmp ./docker-compose.yml
 
-# Write dappnode version
-sed -e "s/\"version\":[ ]\"[0-9]\{1,\}\.[0-9]\{1,\}\.[0-9]\{1,\}\"/\"version\": \"${AVADO_VERSION}\"/" ./dappnode_package.json \
+# Copy sections between *_JSON_EXPORT of docker-compose.yaml to dappnode_package.json
+# and also set dappnode version
+sed -n '/BEGIN_JSON_EXPORT/,/END_JSON_EXPORT/{//!p}' ./docker-compose.yml \
```

Comment:
How about you just use `yq` to transform the yaml into json and then pick whatever you need using `jq`? That way you don't have to do that JSON_EXPORT dance.

Reply:
Because `yq` is an extra dependency just for a singular task, which can be handled using tools we already have/use.